tests/scripts/install_modules.bat

```diff
@@ -0,0 +1,12 @@
+%1 -m pip install -U pytest
```

Comment:
Will it be possible to add some docs/headers about how this particular script works, what arguments are required and what requirements are needed?